src/js/utils/JobUtil.js

```diff
@@ -72,16 +72,22 @@ const JobUtil = {
       cmd: general.cmd,
       cpus: general.cpus,
       mem: general.mem,
-      disk: general.disk
+      disk: general.disk,
+      docker
     });
 
-    if (docker && docker.image) {
-      Object.assign(spec.run, { docker });
-      if (docker_parameters != null && docker_parameters.items != null) {
-        spec.run.docker.parameters = docker_parameters.items;
-      } else {
-        spec.run.docker.parameters = [];
-      }
+    if (docker_parameters != null && docker_parameters.items != null) {
```

Comment:
Why not `!===` ?

Reply:
I always using not the strict comparison for null since it will catch more cases. For example on an object these two cases yield different examples and I want to cover the second case.
```JS
({}).a === null; // false
({}).a == null; // true
```